src/downloadDocumentation.js

```diff
@@ -32,17 +34,10 @@ module.exports = {
   },
 };
 
-function getFileExtension(filename) {
-  const path = require('path');
-  let ext = path.extname(filename || '').split('.');
-
-  return ext[ext.length - 1];
-}
```

Comment:
Moved to the _src/fileUtils.js_ file, to be able to re-use them on the _src/generateDocumentation.js_ one.